### DISCUSSION TITLE
Power/HR Zone bar

### DISCUSSION THREAD

**Discussion by jchinte** [time4tea/gopro-dashboard-overlay#95] [Show and tell]
I made a gradient bar chart with configurable power zones inspired by Norcal Cycling on YouTube.

![image](https://user-images.githubusercontent.com/9040266/215988173-fd4d8db6-205e-4812-adbd-ac5279d95d41.png)

A lot of people train with a 5 or even 7 zone model, but my code is currently hard-coded for 3 zones (not including the lowest)

**Comment by time4tea** [comment]
This looks amazing! Are you OK to share your code? If so, please make a PR, agreeing to the contributor stuff, and I can see about including it in the software....

**Reply by jchinte** [reply]
Would be my first PR! Are you able to walk me through this?

**Reply by time4tea** [reply]
Hiya. The generic instructions are at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

If you made the code changes on a fork of the repo it's just a couple of clicks.

This just helps keep an audit trail of where the code came from and that its authorised to be in the software. 

If you made the code changes on a clone of this repo, then you alternatively could just add a patch file here, along with an affirmation that you agree to Develper Certificate of Origin: https://developercertificate.org/ and that code is OK to be licensed under MIT licence.

Making patch files is documented at https://www.git-tower.com/learn/git/faq/create-and-apply-patch/

Cheers!

**Reply by jchinte** [reply]
I set a pull request,so hopefully that worked!

I didn't include the xml code though I can upload it separately or just paste the relevant code somewhere

**Reply by time4tea** [reply]
Awesome. Thanks for your contribution! I hope to be able to look at this today.

**Reply by time4tea** [reply]
Hi - I took a look at the PR - not quite sure it was for the right files / branch etc. Maybe you can point me to the code you changed? Is it in github?

**Reply by jchinte** [reply]
It is! I think I must have chosen the wrong branch to merge from. Hopefully it's working this time?
